packages/conf-ncurses/conf-ncurses.1/opam

```diff
@@ -7,6 +7,9 @@ license: "MIT"
 build: [
   ["pkg-config" "ncurses"] { os != "darwin"}
 ]
+depends: [
+  "conf-pkg-config" { os != "darwin"}
```

Comment:
I'm fine with both solutions, but just keep in mind that every time you add/remove a dependency to ocamlfind, that triggers a large recompilation for our users, so should pick one solution which works and it well tested :-) following that, just depending on `conf-pkg-config` seems to be the less error-prone solution.